packages/bundler-plugin-core/src/utils/normalizeOptions.ts

```diff
@@ -80,6 +80,23 @@ const optionsSchemaFactory = (options: Options) =>
       .string({ invalid_type_error: "`uploadToken` must be a string." })
       .optional(),
     uploadOverrides: UploadOverridesSchema.optional(),
+    sentry: z
+      .object({
+        sentryOnly: z.boolean({
+          invalid_type_error: "`sentry.sentryOnly` must be a boolean.",
+        }),
+        isEnabled: z.boolean({
+          invalid_type_error: "`sentry.isEnabled` must be a boolean.",
+        }),
+        org: z.string({ invalid_type_error: "`sentry.org` must be a string." }),
+        project: z.string({
+          invalid_type_error: "`sentry.project` must be a string.",
+        }),
+        environment: z.string({
+          invalid_type_error: "`sentry.environment` must be a string.",
+        }),
+      })
+      .optional(),
```

Comment:
@nicholas-codecov do u think it makes more sense to set `process.env.SENTRY_ORG`, `process.env.SENTRY_PROJECT`, etc here, instead of `sendSentryBundleStats` in `sentryUtils.ts`

Reply:
Do you mean to default to these options?

Reply:
I'm not sure if I'm reading this correctly, but these values should be passed through the configuration, so you can grab them off of the options object that is returned by this function, no need to touch `process.env` inside the plugins.

Reply:
yeah I guess if the intention is that this bundler plugin gets consumed by a sentry one, we can get away with always explicitly passing the options.

We'll need `process.env` discovery somewhere though because a ton of people have this setup to make sentry-cli work in their CI pipeline.